src/lib/tabs/tab-body.html

```diff
@@ -0,0 +1,6 @@
+<div class="md-tab-body-content"
+     [@position]="_position"
```

Comment:
Nit: This animation name doesn't really describe what is happening with the animation.